internal/manifests/var.go

```diff
@@ -15,6 +15,12 @@ const (
 	DefaultContainerImage = "docker.io/grafana/loki:2.2.1"
 )
 
+func commonAnnotations(opt Options) map[string]string {
```

Comment:
since `Options` is not needed, can you reduce the complexity of this and change the signature to `commonAnnotations(sha string)`?